applications/tari_console_wallet/src/ui/components/notification_tab.rs

```diff
@@ -4,9 +4,6 @@
 // are cleared.
 // Currently notifications are only added from the wallet_event_monitor which has
 // add_notification method.
-// TODO: auto delete old notifications. #LOGGED
```

Comment:
This TODO is handled by the ability to manually clear the notifications, I don't think its ideal to do it automatically and notifications are not persisted so the manual option seems adequate.